redis/src/aio/tokio.rs

```diff
@@ -124,7 +121,7 @@ impl RedisRuntime for Tokio {
         hostname: &str,
         socket_addr: SocketAddr,
         insecure: bool,
-        _: &Option<TlsConnParams>,
+        params: &Option<TlsConnParams>,
```

Comment:
can't comment on line 119 - shouldn't it also be replaced by `#[cfg(any(feature = "tokio-rustls-comp", feature = "tokio-native-tls-comp"))]`?

Reply:
Right, all of these can use the `tokio-*-comp` feature names. Fixed now. I made the corresponding change in `async_std.rs` too.

There is a separate implementation for `rustls` builds, so the right cfg here is `#[cfg(all(feature = "tokio-native-tls-comp", not(feature = "tokio-rustls-comp")))]`.